src/libraries/Directory.Build.props

```diff
@@ -45,7 +45,7 @@
 
       See Documentation/coding-guidelines/project-guidelines.md for more details on our configurations.
     -->
-    <TargetGroup Condition="'$(TargetGroup)' == ''">netcoreapp</TargetGroup>
+    <TargetGroup Condition="'$(TargetGroup)' == ''">netcoreapp5.0</TargetGroup>
```

Comment:
Shouldn't this be $(NetCoreAppCurrent) as well? Have you checked other places where you are hardcoding netcoreapp5.0?